mixer/cmd/root.go

```diff
@@ -362,6 +211,26 @@ func checkAllDeps() bool {
 	return ok
 }
 
+func containerExecute(cmd *cobra.Command, args []string) error {
+	if builder.Offline {
+		fmt.Println("Warning: Unable to determine upstream format in --offline mode, " +
```

Comment:
For consistency i'd `_, _ = fmt.Fprintf(os.Stderr, ...)` this one so you don't have to pull in the log package just for here.

Reply:
Log is already included in build.go, so I used log instead.